roles/openshift_master/tasks/main.yml

```diff
@@ -242,8 +242,8 @@
 
 - name: Dump logs from master-api if it failed
   command: journalctl --no-pager -n 100 -u {{ openshift_service_type }}-master-api
-  when:
-  - l_start_result is failed
+  # when:
```

Comment:
remove these if unwanted

Reply:
Good catch.  That was in there for testing purposes because I couldn't get the API to start on the in-tree job.

Figured out it was an etcd problem; I didn't use lookup for an environment variable where I should.  I will revert this file as it doesn't need to be changed.